api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -264,4 +295,430 @@ public void saveConcept_shouldForceSetFlagIfSetMembersExist() throws Exception {
 		//Then
 		assertTrue(concept.getSet());
 	}
+	
+	/**
+	 * @see ConceptServiceImpl#purgeConcept(Concept)
+	 * @Verifies should purge the concept if not being used by an obs
+	 */
+	@Test
+	public void purgeConcept_shouldPurgeTheConceptIfNotBeingUsedByAnObs() throws Exception {
+		conceptService.purgeConcept(conceptService.getConcept(88));
+		assertNull(conceptService.getConcept(88));
```

Comment:
@mavrk generally if you reuse numbers/strings/... its good to create a local variable in your test. this way if the id changes in the test data only one change is needed here. this advice can be applied to all your tests